java-aws-lambda-gradle/build.gradle

```diff
@@ -0,0 +1,29 @@
+plugins {
+    id 'java'
+}
+
+sourceCompatibility = JavaVersion.VERSION_11
+targetCompatibility = JavaVersion.VERSION_11
+
+repositories {
+    mavenCentral()
+    maven {
+        url "https://nexus.nuiton.org/nexus/service/local/repositories/thirdparty/content/"
```

Comment:
Write something about this in the readme (or in the gradle and maven build files), gotta explain why you've added this.